migrations/20200824194331-patient-gender.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.bulkUpdate(
+          'patients',
+          { gender: 'F' },
+          {
+            gender: {
+              [Sequelize.Op.notIn]: ['F', 'Female', 'Male', 'M'],
+            },
+          },
+          { transaction: t }
+        ),
+      ])
+        .then(
+          queryInterface.bulkUpdate(
+            'patients',
+            { gender: 'F' },
+            { gender: 'Female' },
+            { transaction: t }
+          ),
+          queryInterface.bulkUpdate(
+            'patients',
+            { gender: 'M' },
+            { gender: 'Male' },
+            { transacton: t }
+          )
+        )
+        .then(() =>
+          queryInterface.changeColumn(
+            'patients',
+            'gender',
+            {
+              type: Sequelize.DataTypes.ENUM('M', 'F'),
+            },
+            {
+              transaction: t,
+            }
+          )
+        );
```

Comment:
- no real point in using `Promise.all` if you only have 1 promise in it
- the 3 updates can be done asynchronously since they should all be mutually exclusive

```suggestion
        queryInterface.bulkUpdate(
          'patients',
          { gender: 'F' },
          { gender: 'Female' },
          { transaction: t }
        ),
        queryInterface.bulkUpdate(
          'patients',
          { gender: 'M' },
          { gender: 'Male' },
          { transacton: t }
        ),
      ]).then(() =>
        queryInterface.changeColumn(
          'patients',
          'gender',
          {
            type: Sequelize.DataTypes.ENUM('M', 'F'),
          },
          {
            transaction: t,
          }
        )
      );
```